projects/js-packages/components/components/record-meter-bar/test/component.tsx

```diff
@@ -0,0 +1,60 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+import { render } from '@testing-library/react';
+
+/**
+ * Internal dependencies
+ */
+import RecordMeterBar, { RecordMeterBarProps } from '../index';
+
+describe( 'RecordMeterBar', () => {
+	const testProps: RecordMeterBarProps = {
+		items: [
+			{ count: 18, label: 'Posts', backgroundColor: '#00BA37' },
+			{ count: 30, label: 'Plugins', backgroundColor: '#3895BA' },
+			{ count: 52, label: 'Comments', backgroundColor: '#E68B28' },
+			{ count: 24, label: 'Authors', backgroundColor: '#3859BA' },
+		],
+	};
+	it( 'renders nothing when no items are passed', () => {
+		const { container } = render( <RecordMeterBar items={ [] } /> );
+
+		expect( container.querySelector( '.record-meter-bar__items' ).children.length ).to.be.equal(
+			0
+		);
```

Comment:
Using `querySelector` is not the best approach to select items into `RTL`, you should use queries for it: https://testing-library.com/docs/queries/about

Reply:
Oh thanks! But I don't know which query will help me select the `div` with classname `record-meter-bar__items`.

Here is the actual output:

```html
<div>
  <div class="record-meter-bar">
    <div class="record-meter-bar__items"></div>
    <div class="record-meter-bar__legend">
      <ul class="record-meter-bar__legend--items"></ul>
    </div>
  </div>
</div>

```